app/javascript/components/enrollment/Enrollment.js

```diff
@@ -84,6 +84,9 @@ class Enrollment extends React.Component {
     window.onbeforeunload = null;
     axios.defaults.headers.common['X-CSRF-Token'] = this.props.authenticity_token;
 
+    // this.state.enrollmentState.patient.jurisdiction_id is a string while this.props.patient.jurisdiction_id is a number so it was always triggering
+    this.props.patient.jurisdiction_id = this.props.patient.jurisdiction_id.toString();
```

Comment:
I'm not sure I'm following this change. This _should_ be an integer, so why do we need to change the prop to a string? I'd rather convert the strings to ints elsewhere (which I think we do with `parseInt() calls).

Reply:
I updated setEnrollmentState to save jurisdiction_id as a number